dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/ajax/ContentletAjax.java

```diff
@@ -629,6 +629,8 @@ public List searchContentletsByUser(List<BaseContentType> types, String structur
 		            break;
 		        }
 		    }
+
+			luceneQuery.append("+" + ESMappingConstants.SYSTEM_TYPE + ":false ");

```

Comment:
I think you need to add a whitespace after this query sentence, otherwise the following appends will be broken